appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -1098,3 +1174,4 @@ public void execute() {
   }
 
 }
+
```

Comment:
You don't need the extra line here, but make sure that there's a newline at the end of the file if you remove it.